sdk/storage/storage-blob/README.md

```diff
@@ -114,62 +117,31 @@ For example, you can create following CORS settings for debugging. But please cu
 - Exposed headers: \*
 - Maximum age (seconds): 86400
 
-## SDK Architecture
-
-The Azure Storage SDK for JavaScript provides low-level and high-level APIs.
+## Examples
 
-- BlobServiceClient, ContainerClient and BlobClient objects provide the low-level API functionality and map one-to-one to the [Azure Storage Blob REST APIs](https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-rest-api).
+### Create the Blob service Client
```

Comment:
`Blob service Client` 
Casing - `Blob Service Client`

Reply:
readme template recommends sentence capitalization.

---
In reply to: [295982035](https://github.com/Azure/azure-sdk-for-js/pull/3744#discussion_r295982035) [](ancestors = 295982035)

Reply:
If this is the design, the design itself is bad ? 🤔